knowledge-base/numerictextbox-disable-arrows.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Disabling NumericTextBox Arrows at Min or Max Value in Blazor
+description: Learn how to programmatically disable the increase or decrease arrows of a NumericTextBox in Blazor when the value reaches its minimum or maximum limit.
+type: how-to
+page_title: How to Disable NumericTextBox Arrows on Min or Max Value in Blazor
+slug: numerictextbox-disable-arrows
+tags: numerictextbox, blazor, disable arrows, conditional styling
+res_type: kb
+ticketid: 1665216
+---
+
+## Environment
+
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>NumericTextBox for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+When using the NumericTextBox with arrows enabled, I want to disable the down arrow programmatically at the component's minimum value to prevent looping to the maximum value. 
```

Comment:
```suggestion
In a NumericTextBox with enabled arrows, I want to disable the down arrow programmatically at the component's minimum value to prevent looping to the maximum value. 
```